.github/workflows/azure-cd.yml

```diff
@@ -0,0 +1,63 @@
+# This is just an example workflow for continuous deployment.
+# You should customize it to meet your own requirements.
+name: "Continuous Deployment"
+on:
+  # When new commits are pushed onto the main branch.
+  push:
+    branches:
+      - master
+jobs:
+  buildAndPublish:
+    runs-on: ubuntu-latest
+    # You can uncomment the line below to use environments (refer to https://docs.github.com/en/actions/reference/environments).
+    #environment: test_environment
```

Comment:
I think it would be beneficial to merge this job and [.github/workflows/github-ci-beaverhours.yml](https://github.com/beaver-hours/BeaverHours/blob/8f47b4f4e76cd2b6bd830eadfce344a769de482b/.github/workflows/github-ci-beaverhours.yml) as both the only thing in that file that's not in this one is checking the code formatting.

Reply:
I'm imagining a situation where we add a bit more to the CI workflow later. E.g. Adding linting or other testing? I don't see too much of a downside or upside either way.

Reply:
I think testing is a solid reason to keep them separated at this point, looks good to me!